### PR TITLE
update manualBootstrap.md for AIX

### PR DESCRIPTION
**Pull request by gdams**
Before Landing:

- [x] @mhdawson to upload all the artifacts to  https://ci.nodejs.org/downloads/aix/

### CODE REVIEW

File: setup/aix61/manualBootstrap.md [hunk 1]
```diff
@@ -98,6 +121,13 @@ LIBPATH=/usr/lib curl https://bootstrap.pypa.io/get-pip.py | python
 ln -s /opt/freeware/bin/pip /usr/bin/pip
 ```
 
+#### tap2junit
+
+```bash
+pip install tap2junit
```
Comment (refack) [inline setup/aix61/manualBootstrap.md]:
```suggestion
python -m pip install --upgrade pip pipenv git+https://github.com/refack/tap2junit.git
```

Comment (richardlau) [reply setup/aix61/manualBootstrap.md]: Why not https://github.com/jbergstroem/tap2junit? (Or perhaps tap2junit should be moved under the foundation?)

Comment (refack) [reply setup/aix61/manualBootstrap.md]: I made a few bug fixes https://github.com/refack/tap2junit/commit/7d9574915148b488b23be699f2a97f6eba509160

Comment (richardlau) [reply setup/aix61/manualBootstrap.md]: https://github.com/jbergstroem/tap2junit/commit/7d9574915148b488b23be699f2a97f6eba509160

**Pull request merged**